Remotion/Web/Core/WebString.cs

```diff
@@ -176,6 +176,23 @@ public void AddAttribute ([NotNull] HtmlTextWriter writer, [NotNull] string attr
       }
     }
 
+    /// <summary>
+    /// Tries to convert this <see cref="WebString"/> into a <see cref="PlainTextString"/>.
+    /// Only works if <see cref="Type"/> is <see cref="WebStringType"/>.<see cref="WebStringType.PlainText"/>,
+    /// and will throw an <see cref="InvalidOperationException"/> otherwise.
```

Comment:
There's too much redundant information in the summary (i.e. the exception info). If possible, the summary should be a single line with just the happy path semantics.
```suggestion
    /// Converts this <see cref="WebString"/> into a <see cref="PlainTextString"/> if <see cref="Type"/> is <see cref="WebStringType.PlainText"/>.
```